docs/zreference.bib

```diff
@@ -39,4 +39,4 @@ @article{zadeh2013molecular
   pages={927--936},
   year={2013},
   publisher={Springer}
-}
+}
```

Comment:
I would add a new line at the end of the file for consistency.

Reply:
Newline has been added, 9195b51. It's good to follow a good convention. Thanks.